src/herbpy/herbrobot.py

```diff
@@ -271,7 +270,72 @@ def SetStiffness(self, stiffness):
         self.head.SetStiffness(stiffness)
         self.left_arm.SetStiffness(stiffness)
         self.right_arm.SetStiffness(stiffness)
-
+        
+    def DetectHuman(self, env, orhuman=2, load_hum=True, 
+                    hrc=False, continuous=True,      
+                    herb_sim=True, enable_legs=True,             
+                    segway_sim=True, action='stamp'):
+        """Use the kinbody detector to detect objects and add
+        them to the environment
+        """
+        import rospy
+        from tf import TransformListener, transformations
+
+        node_name = 'detecthuman'  
+
+        if orhuman==0: 
```

Comment:
What is the meaning of the `orhuman` argument? It looks like an enum that controls the method used to track the skeleton. If that is the case, I suggest using a meaningful string for each choice instead of a number.